scheduler/config.edn

```diff
@@ -24,6 +24,20 @@
                   :executable false
                   :extract true
                   :value #config/env "COOK_EXECUTOR"}}
+ ;; To configure a hook, define a factory function and its arguments.
+ ;; Needed for running hook integration tests outside of travis environment.
+ :submission-hook {:batch-timeout-seconds 40
+                   :factory-fn "cook.demo-plugin/submission-factory"
+                   :arguments {:launch-url #config/env "COOK_DEMO_HOOKS_LAUNCH_URL"
+                               :submit-url #config/env "COOK_DEMO_HOOKS_SUBMIT_URL"}}
+ ;; To configure a hook, define a factory function and its arguments.
+ ;; Needed for running hook integration tests outside of travis environment.
+ :launch-hook {:age-out-last-seen-deadline-minutes 10
+               :age-out-first-seen-deadline-minutes 600
+               :age-out-seen-count 10
+               :factory-fn "cook.demo-plugin/launch-factory"
+               :arguments {:launch-url #config/env "COOK_DEMO_HOOKS_LAUNCH_URL"
+                           :submit-url #config/env "COOK_DEMO_HOOKS_SUBMIT_URL"}}
```

Comment:
I thought we were going to do:

```clojure
:plugins {:job-submission-validator {:factory-fn ...
                                     :arguments ...}
          :job-launch-filter {:factory-fn ...
                              :arguments ...}}
```

Questions/comments:

- top-level `:plugins` section?
- `:job-submission-validator` instead of `:submission-hook`
- `:job-launch-filter` instead of `:launch-hook`
- why do they both have both a `:launch-url` and a `:submit-url`?

Reply:
Config structure changes made. (And, good catch on the duplicate URL's)